styles.css

```diff
@@ -0,0 +1,427 @@
+}
+
+.header-checkout a:link {
+  text-decoration: none;
+  padding: 4px 8px 4px 8px;
+  background-color: #9bc344;
+  border-radius: 1px;
+  border: none;
+  width: 240px;
+  color: #fff;
+}
+
+.header-checkout a:visited {
+  color: #fff;
+  background-color: #9bc344;
+}
+
+.header-checkout a:hover {
+  color: #fff;
+  background-color: #6a872c;
+}
+
+.header-checkout a:active {
+  color: #fff;
+  background-color: #103861;
+}
+
+nav {
+  background-color: #D3D5D3;
+  margin-top: 0px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  font-family: monospace;
+  padding-left: 35px;
+  font-size: 25px;
+
+}
+nav a {
+  text-decoration: none;
+  color: #103861;
+}
+
+nav a:hover {
+  text-decoration: underline;
+}
+.container {
+  margin-left: 60px;
+}
+h2 {
+  font-family: serif;
+  font-weight: 700;
+  font-size: 2em;
+  margin-top: 40px;
+  color: #103861;
+}
+
+h3 {
+  font-family: sans-serif;
+  font-weight: 700;
+  font-size: 1em;
+  color: #103861;
+}
+
+h4 {
+  font-family: serif;
+  font-weight: 700;
+  font-size: 1em;
+  color: #103861;
+}
+
+h5 {
+  font-family: serif;
+  font-weight: 700;
+  font-size: 1em;
+  color: #103861;
+}
+
+p {
+  font-family: sans-serif;
+  font-weight: 300;
+  font-size: 1em;
+}
+
+.home-container {
+  margin: 20px 10px 40px 0px;
+  max-width: 90%;
+}
+
+.home-container img {
+  float: left;
+  padding-right: 40px;
+}
+
+.home-container h2 p {
+  float: right;
+  padding: 10px 10px 80px 0px;
+}
+
+.dog-profiles {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  clear: both;
+  max-width: 90%;
+  padding-top: 20px;
+}
+
+.dog-profiles section {
+  border: 1px solid #ccc;
+  width: 200px;
+  margin: 10px 30px 20px 0px;
+}
+
+.dog-profiles a:link {
+  font-family: sans-serif;
+  font-weight: 700;
+  font-size: .7em;
+  margin-top: 20px;
+  margin-bottom: 0px;
+  padding: 4px 70px 4px 62px;
+  background-color: #9bc344;
+  border-radius: 1px;
+  text-decoration: none;
+  color: #fff;
+}
+
+.dog-profiles a:visited {
+  color: #fff;
+  background-color: #9bc344;
+}
+
+.dog-profiles a:hover {
+  color: #fff;
+  background-color: #6a872c;
+}
+
+.dog-profiles a:active {
+  color: #fff;
+  background-color: #103861;
+}
+
+.dog-info {
+  padding: 0px 8px 10px 8px;
+}
+
+.see-more {
+  max-width: 100%;
+  height: 80px;
+  padding: 40px;
+}
+
+.see-more a:link {
+  font-family: 'Public Sans', sans-serif;
+  font-weight: 700;
+  font-size: .7em;
+  padding: 8px 40px 8px 40px;
+  margin-right:45%;
+  margin-left: 35%;
+  background-color: #ccc;
+  border-radius: 1px;
+  text-decoration: none;
+  color: green;
+}
+
+.see-more a:visited {
+  color: #fff;
+  background-color: #9bc344;
+}
+
+.see-more a:hover {
+  color: #fff;
+  background-color: #6a872c;
+}
+
+.see-more a:active {
+  color: #fff;
+  background-color: #103861;
+}
+
+.blog-article {
+  display: flex;
+  flex-flow: row wrap;
+  margin: 0px 10px 40px 0px;
+  max-width: 90%;
+}
+
+.blog-article img {
+  float: left;
+}
+
+.blog-text {
```

Comment:
Should be using margins instead of padding, margins are external while padding is internal. Also position absolute is not a great choice. since you are floating the image left, the text should normally appear right and take up remaining space.

Reply:
I don't know what's happen to my code. If I don't use absolute, the content will go right, but under the picture. I tried so many ways, only the absolute can make the text stay next to the picture.